ui/components/multichain/nft-item/nft-item.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import NftDefaultImage from '../../app/nft-default-image/nft-default-image';
+import {
+  AvatarNetwork,
+  BadgeWrapper,
+  BadgeWrapperAnchorElementShape,
+} from '../../component-library';
+import {
+  BackgroundColor,
+  BorderRadius,
+  Display,
+  Size,
+} from '../../../helpers/constants/design-system';
+import Box from '../../ui/box/box';
+
+export const NftItem = ({
+  alt,
+  name,
+  src,
+  networkName,
+  networkSrc,
+  tokenId,
+  onClick,
+}) => {
+  return (
+    <Box data-testid="nft-item">
+      <BadgeWrapper
+        badge={
+          <AvatarNetwork
+            size={Size.SM}
+            name={networkName}
+            src={networkSrc}
+            borderColor={BackgroundColor.backgroundDefault}
+            borderWidth={2}
+            data-testid="nft-network-badge"
+            className="nft-network-badge"
+          />
+        }
+        anchorElementShape={BadgeWrapperAnchorElementShape.rectangular}
+        positionObj={{ top: -4, right: -4 }}
+        display={Display.BLOCK}
+      >
+        {src ? (
+          <Box
+            as="button"
+            className="nfts-items__item"
+            borderRadius={BorderRadius.LG}
+            onClick={onClick}
+          >
+            <img
+              className="nfts-items__item-image"
+              data-testid="nft-image"
+              style={{ borderRadius: 8 }}
```

Comment:
So it does work but only because React JSX assumes pixels as a unit for some css properties.
However it's always good to specify units. I'll get that changed, thanks

Reply:
Since this has a `classname`, we should be able to avoid `style={{}}}` and just put it in the spreadsheet